examples/src/app.module.css

```diff
@@ -0,0 +1,78 @@
+html {
+  font-family: 'Roboto', sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+h3 {
+  margin-bottom: 0;
+  text-align: center;
+}
+
+h5 {
+  margin-top: 0;
+}
+
+.testApp {
```

Comment:
Consider keeping the class name single worded to avoid casing issues. It is more common to use dash-separated class names instead of camel-cased.